src/java.base/share/classes/jdk/crac/impl/ExceptionHolder.java

```diff
@@ -23,16 +23,25 @@ public void throwIfAny() throws E {
         }
     }
 
+    public boolean hasException() {
+        return exception != null;
+    }
+
+    public void reSuppress(Exception e) {
```

Comment:
I don't like the suggestion. What is the problem with the original name?

Reply:
OK. It's just unclear, when reading the use `re` would be some acronym (just related to **R**estore**E**xceptions), otherwise it doesn't need the camelcasing (https://en.wiktionary.org/wiki/resuppress). Forget about it if you're satisfied with the current one.